### PR TITLE
Upgrade to useDApps/core 1.2.11

### PR DESCRIPTION
**Pull request by wagyu298**
Reduce `eth_chainId` API call by using StaticJsonRpcProvider.
useDApps/core 1.0.8 or later uses StaticJsonRpcProvider for readOnlyUrls.
See https://github.com/ethers-io/ethers.js/issues/901 for more details.

### CODE REVIEW

File: packages/nouns-webapp/package.json [hunk 1]
```diff
@@ -59,8 +59,7 @@
     "redux": "^4.1.0",
     "redux-devtools-extension": "^2.13.9",
     "remark-breaks": "^3.0.1",
-    "web-vitals": "^1.0.1",
-    "react-transition-group": "^4.4.5"
```
Comment (ms76) [inline packages/nouns-webapp/package.json:63]: 📝 Duplicate L58

File: packages/nouns-webapp/src/wrappers/nounsAuction.ts [hunk 2]
```diff
@@ -94,7 +90,11 @@ export const useSettleCurrentAndCreateNewAuction = () => {
...
   return {
     send: async (...args: any[]): Promise<void> => {
-      const contract = connectContractToSigner(nounsAuctionHouseContract, undefined, library);
+      const contract = connectContractToSigner(
+        nounsAuctionHouseContract,
+        undefined,
+        library && 'getSigner' in library ? library.getSigner() : undefined,
+      );
       const gasLimit = await contract.estimateGas.settleCurrentAndCreateNewAuction(...args);
       settleAuction(...args, {
```
Comment (ms76) [inline packages/nouns-webapp/src/wrappers/nounsAuction.ts]: await is missing?

**Review by ms76 — COMMENTED** [review]
I’ve checked your fix. There were 2 points where I think await is missing, although they are not part of this fix.

One is commented and the other is below.
https://github.com/CNNouns/cnnouns-monorepo/pull/23/files#diff-aefeb8faf6b9330978ae529f732f9d79575267e925c501f429f462be0b76bab8R137

**Comment by wagyu298** [comment]
@ms76 Thank you for your code reviews! I have fixed the pointed out codes.
After that, I got `yarn install` error on CI tests. I solve the problem by other pull request #25.
Could you review the pull request #25 first?

**Review by ms76 — APPROVED** [review]
@wagyu298 I have reviewed your fix. LGTM!

**Comment by wagyu298** [comment]
#25 merged, test passed.

**Comment by wagyu298** [comment]
@ms76 Thank you for your code reviews!

**Pull request merged**